pkgs/applications/networking/instant-messengers/element/element-desktop-package.json

```diff
@@ -2,7 +2,7 @@
   "name": "element-desktop",
   "productName": "Element",
   "main": "lib/electron-main.js",
-  "version": "1.8.2",
+  "version": "1.8.4",
```

Comment:
It's fairly weird that nothing else in here has changed. Investigating.

Reply:
OK, doesn't matter here, matrix-js-sdk is directly pulled from `pkgs.element-web` where I just confirmed that it got updated.